RockLib.Logging/RockLib.Logging.csproj

```diff
@@ -24,13 +24,14 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="RockLib.Configuration" Version="2.3.0" />
-    <PackageReference Include="RockLib.Configuration.ObjectFactory" Version="1.2.2" />
+    <PackageReference Include="RockLib.Configuration" Version="2.3.1" />
+    <PackageReference Include="RockLib.Configuration.ObjectFactory" Version="1.4.0" />
     <PackageReference Include="RockLib.Diagnostics" Version="1.0.1" />
+    <PackageReference Include="System.ValueTuple" Version="4.5.0" />
```

Comment:
Probably didn't mean to add this (should only need to update the package in the conditional section below).

Reply:
Ah it probably did that when I tried to update via the package manager first and it failed.  Good catch.